src/frontend/components/DashboardDocument/index.spec.tsx

```diff
@@ -0,0 +1,182 @@
+import { render, wait } from '@testing-library/react';
```

Comment:
in commit message

`Document need ` => `Documents need`
`it can be splitted later` => `it can be split later`